package.json

```diff
@@ -30,7 +30,7 @@
   "dependencies": {
     "assert-plus": "^1.0.0",
     "jsprim": "^1.2.2",
-    "sshpk": "^1.7.0"
+    "sshpk": "^1.14.1"
```

Comment:
This should now be `^1.14.2` to account for the `new Buffer` usage fix. See https://github.com/joyent/node-sshpk/pull/46

Reply:
Done, resubmitted pull request with 1.14.2

Reply:
What's the hold up?

Reply:
Sorry for that @BYK
@knoxcard i'm working on fixing snyk vulnerabilities generated by `sshpk` version 1.7 that seems to be fixed on newest version 1.14.2. 
I was wondering if there is a release date with that package updated.
Thanks!!